tests/components/unifi/test_sensor.py

```diff
@@ -74,31 +81,39 @@ async def test_sensors(hass):
         hass,
         options={
             CONF_ALLOW_BANDWIDTH_SENSORS: True,
+            CONF_ALLOW_UPTIME_SENSORS: True,
             CONF_TRACK_CLIENTS: False,
             CONF_TRACK_DEVICES: False,
         },
         clients_response=CLIENTS,
     )
 
     assert len(controller.mock_requests) == 4
-    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 4
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 6
 
     wired_client_rx = hass.states.get("sensor.wired_client_name_rx")
     assert wired_client_rx.state == "1234.0"
 
     wired_client_tx = hass.states.get("sensor.wired_client_name_tx")
     assert wired_client_tx.state == "5678.0"
 
+    wired_client_uptime = hass.states.get("sensor.wired_client_name_uptime")
+    assert wired_client_uptime.state == "2020-09-14T14:41:45+00:00"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)